src/setup/run.go

```diff
@@ -88,55 +90,95 @@ func ProvisionFunctions(config Configuration) {
 
 // ProvisionFunctionsServerless will deploy, reconfigure, etc. functions to get ready for the sub-experiments.
 func ProvisionFunctionsServerless(config *Configuration, serverlessDirPath string) {
+	switch config.Provider {
+	case "aws":
+		ProvisionFunctionsServerlessAws(config, serverlessDirPath)
+	case "azure":
+		ProvisionFunctionsServerlessAzure(config, serverlessDirPath)
+	case "gcr":
+		ProvisionFunctionsGCR(config, serverlessDirPath)
+	}
+}
+
+// ProvisionFunctionsServerlessAws will deploy, reconfigure, etc. functions to get ready for the sub-experiments.
+func ProvisionFunctionsServerlessAws(config *Configuration, serverlessDirPath string) {
```

Comment:
```suggestion
func ProvisionFunctionsServerlessAWS(config *Configuration, serverlessDirPath string) {
```
Let's rename this for consistency of the word "AWS" (capitalized) with other functions as well